element_array_ephys/ephys_organoids.py

```diff
@@ -128,382 +129,180 @@ class AcquisitionSoftware(dj.Lookup):
-            ("*.oebin", "Open Ephys"),
-        ):
-            ephys_meta_filepaths = list(session_dir.rglob(ephys_pattern))
-            if ephys_meta_filepaths:
-                acq_software = ephys_acq_type
-                break
-        else:
-            raise FileNotFoundError(
-                f"Ephys recording data not found!"
-                f" Neither SpikeGLX nor Open Ephys recording files found"
-                f" in {session_dir}"
-            )
-
-        supported_probe_types = probe.ProbeType.fetch("probe_type")
-
-        if acq_software == "SpikeGLX":
-            for meta_filepath in ephys_meta_filepaths:
-                spikeglx_meta = spikeglx.SpikeGLXMeta(meta_filepath)
-                if str(spikeglx_meta.probe_SN) == inserted_probe_serial_number:
-                    break
-            else:
-                raise FileNotFoundError(
-                    f"No SpikeGLX data found for probe insertion: {key}"
-                    + " The probe serial number does not match."
-                )
-
-            if spikeglx_meta.probe_model in supported_probe_types:
-                probe_type = spikeglx_meta.probe_model
-                electrode_query = probe.ProbeType.Electrode & {"probe_type": probe_type}
-
-                probe_electrodes = {
-                    (shank, shank_col, shank_row): key
-                    for key, shank, shank_col, shank_row in zip(
-                        *electrode_query.fetch("KEY", "shank", "shank_col", "shank_row")
-                    )
-                }
-
-                electrode_group_members = [
-                    probe_electrodes[(shank, shank_col, shank_row)]
-                    for shank, shank_col, shank_row, _ in spikeglx_meta.shankmap["data"]
-                ]
-            else:
-                raise NotImplementedError(
-                    "Processing for neuropixels probe model"
-                    " {} not yet implemented".format(spikeglx_meta.probe_model)
-                )
+        for file in sorted(list(data_files)):
+            start_time = re.search(r".*_(\d{6}_\d{6})", file.stem).groups()[0]
+            start_time = np.datetime64(
+                datetime.strptime(start_time, "%y%m%d_%H%M%S")
+            )  # start time based on the file name
 
             self.insert1(
                 {
                     **key,
-                    **generate_electrode_config(probe_type, electrode_group_members),
-                    "acq_software": acq_software,
-                    "sampling_rate": spikeglx_meta.meta["imSampRate"],
-                    "recording_datetime": spikeglx_meta.recording_time,
-                    "recording_duration": (
-                        spikeglx_meta.recording_duration
-                        or spikeglx.retrieve_recording_duration(meta_filepath)
-                    ),
+                    "start_time": start_time,
+                    "file_name": file.relative_to(session_dir),
+                    "file_path": file,
```

Comment:
I could be mistaken, but this will fail if run more than once on the `subject_directory`.

Reply:
To clarify, we need this table to be run daily, but I don't think it will be based on the `key_source`.  Should this be a `Manual` table and we would search the `subject_directory` daily in the Workflow and insert the files that do not exist in the table?

Reply:
@kabilar. Yes, this assumes that all data will be fully uploaded at once so it could be problematic if they add more dataset to the folder later.  One workaround will be to populate this table using an external raw function (e.g., `ingest_rawdata`)